docs/research.md

```diff
@@ -7,6 +7,30 @@ Following is a list of research papers that have been published using the PDEAre
 If you have used PDEArena in your research, and would like it listed here, please add your paper to [this file](https://github.com/microsoft/pdearena/blob/main/docs/research.yml) by sending a pull request to the [PDEArena repository](https://github.com/microsoft/pdearena).
 
 
+<figure markdown>
```

Comment:
This is auto generated based on https://github.com/microsoft/pdearena/blob/main/docs/research.yml. So better to modify that instead.

Reply:
Ah, thanks! Somehow had missed that file. Moved the paper addition to the yaml file now.